arroyo-console/src/routes/pipelines/JobGraph.tsx

```diff
@@ -9,13 +9,31 @@ import {
   JobMetricsResp,
 } from "../../gen/api_pb";
 
+
+function getNodeColor(backpressure: number): string {
+  // backPressure should be a value between 0 and 100
+
+  const thresholds = [33, 66];
```

Comment:
Something isn't working quite right here -- even when the pipeline is fully backpressured (i.e., the tx queues are at 0 capacity) I don't see any colors

Reply:
BTW, if you want to test backpressure, the easiest way to simulate it is to add a sleep in an operator:

```
tokio::time::sleep(Duration::from_secs(1).await
```

Reply:
This is fixed now. I changed the logic for getting the current backpressure.